client/reader/start/style.scss

```diff
@@ -0,0 +1,281 @@
+
+  	header {
+
+		margin-top: -38px;
+  		text-align: center;
+
+		h1 {
+  			font-size: 1.5em;
+  			font-weight: 500;
+  		}
+
+  		p {
+  			color: $gray;
+  			font-size: 12px;
+  		}
+
+  		.site-icon {
+  			margin: auto;
+  		}
+  	}
+
+  	footer {
+
+  		.reader-start-card__follower-count {
+  			color: $gray;
+  			display: inline-block;
+  			float: left;
+  			font-size: 12px;
+  			font-weight: 600;
+  			margin-top: 20px;
+
+  			@include breakpoint( ">480px" ) {
+  				margin-top: 30px;
+  			}
+  		}
+
+  		.follow-button {
+
+  			float: right;
+  			margin-top: 10px;
+  			right: 0;
+
+			@include breakpoint( ">480px" ) {
+  				margin-top: 20px;
+			}
+
+  			&:last-child {
+  				margin-bottom: -6px;
+  			}
+
+			.gridicon {
+				position: absolute;
+					left: 15px;
+					top: 10px;
+			}
+
+			.follow-button__label {
+				color: $gray-dark;
+				font-size: 16px;
+
+				@include breakpoint( "<660px" ) {
+					display: none;
+				}
+			}
+
+			&:hover {
+				color: $gray-dark;
+
+				.gridicon {
+					fill: $gray-dark;
+				}
+
+				.follow-button__label {
+					cursor: pointer;
+					color: $gray-dark;
+				}
+			}
+		}
+
+		.follow-button.is-following {
+			.follow-button__label {
+				color: $alert-green;
+			}
+
+			&:hover {
+				background: none;
+
+				.gridicon {
+					fill: $alert-green;
+				}
+
+				.follow-button__label {
+					cursor: pointer;
+					color: $alert-green;
+				}
+			}
+		}
+  	}
+}
+
+.reader-start-post-preview {
+	border: 1px solid lighten( $gray, 30% );
+	border-radius: 1px;
+	margin-top: -8px;
+	padding: 10px 10px 0;
+
+	h1 {
+		font-family: 'Merriweather', serif;
+		font-size: 16px;
+		font-weight: 600;
+	}
+
+	p {
+		font-family: 'Merriweather', serif;
+		font-size: 12px;
+		margin: 10px 0 -10px 0;
+
+		&:after {
+			@include long-content-fade( $color: $white, $direction: right, $size: 100% );
+			height: 20px;
+			position: relative;
+				top: -20px;
+		}
+	}
+
+	.reader-start-post-preview__byline {
+		color: $gray;
+		font-size: 12px;
+		margin-bottom: 10px;
+
+		.gravatar {
+			float: left;
+		}
+
+		.reader-start-post-preview__author {
+			margin-left: 4px;
+		}
+	}
+}
+
+.reader-start-footer {
+    animation-duration: 1.3s;
+    animation-fill-mode: forwards;
+    animation-iteration-count: once;
+    animation-name: slideout-bar;
+    animation-timing-function: ease-out;
+	background: rgba( 102, 142, 170, .9 );
+	padding: 8px;
+    position: fixed;
+   		bottom: 0;
+    width: 100%;
+    z-index: 5000;
+
+    .reader-start-footer__action {
+    	color: $white;
+    	margin-left: 36%;
+    	width: 50%;
+
+    	@include breakpoint( "<960px" ) {
+    		margin-left: 10%;
+    		width: 80%;
+    	}
+
+    	@include breakpoint( "<660px" ) {
+    		text-align: center;
+    		width: 90%;
+    	}
+
+    	@include breakpoint( "<480px" ) {
+    		margin: -10px 0 0 10px;
+    	}
+    }
+
+    span {
+    	display: inline-block;
+    	font-size: 18px;
+    	margin-top: 8px;
+    }
+
+    button {
```

Comment:
What is the component that requires this? If you use `Button` it shouldn't be needed.